src/Conversation.php

```diff
@@ -245,7 +245,7 @@ public function addMessage($message, $type = "message")
             )
         );
 
-        $this->last_message = end($this->messages[]);
+        $this->last_message = end($this->messages);
```

Comment:
`Returns the value of the last element or FALSE for empty array.`
faudrait ptet tester s'il est empty avant le `end()`